Localize/lang/strings.json

```
@@ -795,6 +795,7 @@
   "W+mUyI": "Next",
   "W070M2": "of {max}",
   "W6FdMh": "Required. The name of the new property.",
+  "W7kvPF": "The expression is invalid. Please use single quotes.",
```

This scenario shouldn't use ["please"](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please): 

```suggestion
  "W7kvPF": "The expression is invalid. Make sure to use single quotes.",
```

---

done!